gateway/src/node/statistics/sessions.rs

```diff
@@ -1,23 +1,92 @@
 // Copyright 2022 - Nym Technologies SA <contact@nymtech.net>
 // SPDX-License-Identifier: GPL-3.0-only
 
+use nym_credentials_interface::TicketType;
 use nym_node_http_api::state::metrics::SharedSessionStats;
 use nym_sphinx::DestinationAddressBytes;
 use std::collections::{HashMap, HashSet};
 use time::{Date, OffsetDateTime};
 
 use nym_statistics_common::events::SessionEvent;
 
-type SessionDuration = u64; //in miliseconds
+#[derive(PartialEq)]
+enum SessionType {
+    Vpn,
+    Mixnet,
+    Unknown,
+}
+
+impl SessionType {
+    fn to_string(&self) -> &str {
+        match self {
+            Self::Vpn => "vpn",
+            Self::Mixnet => "mixnet",
+            Self::Unknown => "unknown",
+        }
+    }
+}
+
+impl From<TicketType> for SessionType {
+    fn from(value: TicketType) -> Self {
+        match value {
+            TicketType::V1MixnetEntry => Self::Mixnet,
+            TicketType::V1MixnetExit => Self::Mixnet,
+            TicketType::V1WireguardEntry => Self::Vpn,
+            TicketType::V1WireguardExit => Self::Vpn,
+        }
+    }
+}
+
+struct FinishedSession {
+    duration: u64, //in milliseconds,
```

Comment:
imho whenever a field or variable has a unit like seconds, ms, minutes, and is not strongly types, it's preferable to include it in the name like `duration_ms`.

That said, can we change it into a `Duration` instead?

Reply:
Changed in eac43af